api/horizon/resources/operations/object/extend-footprint-ttl.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Extend Footprint TTL Object
+order: 230
+---
+
+import { ExampleResponse } from "@site/src/components/ExampleResponse";
+import { AttributeTable } from "@site/src/components/AttributeTable";
+
+Extends the [TTL](https://soroban.stellar.org/docs/soroban-internals/state-archival#extendfootprintttlop) for a given set of ledger entries.
```

Comment:
You could add cross-references to the operation in the [List of Operations](https://developers.stellar.org/docs/fundamentals-and-concepts/list-of-operations#extend-footprint-ttl), too!